luda-adventure/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
```

Comment:
Please add Ci test for adventure

Reply:
[e100e0c](https://github.com/NamseEnt/namseent/pull/359/commits/e100e0cc5e14acffcf50803dcc0f047c3fdebfd4)